### DISCUSSION TITLE
Strimzipodset logs

### DISCUSSION THREAD

**Discussion by kreuzj** [strimzi/strimzi-kafka-operator#8279] [Q&A]
Hi there,
   is it possible to show logs (agregated) for all pods running in strimzipodset? When Strimzi was running as statefulset it was simply possible to press "l" (when using K9s) and agregated log for all pods running in the sts was displayed. This feature is sometime handy in order to troubleshoot some issues. Didn't find a way to do the same with strimzipodset... Did I miss something?

Thanks
Jan

**Comment by scholzj** [comment]
I'm using a tool called [stern](https://github.com/stern/stern) for watching logs of multiple pods.
